receiver/dockerstatsreceiver/config.go

```diff
@@ -60,7 +60,7 @@ type Config struct {
 	DockerAPIVersion float64 `mapstructure:"api_version"`
 
 	// Metrics config. Enable or disable stats by name.
-	MetricsConfig metadata.MetricsSettings `mapstructure:"stats"`
+	MetricsConfig metadata.MetricsSettings `mapstructure:"metrics"`
```

Comment:
Is it okay to assume those who were using the feature gate to fix this? 
Is there a way that `mapstructure` can do some field aliasing?

Reply:
> Is it okay to assume those who were using the feature gate to fix this?

I think so, yes. It's a feature that's still under development and should expect breaking changes. Once it becomes default, that's a different story.